src/components/Footer.test.jsx

```diff
@@ -27,27 +27,25 @@ const FooterWithContext = ({ locale = 'es' }) => {
   );
 };
 
-const FooterWithLanguageSelector = ({ languageSelected = () => {} }) => {
+const { LANGUAGE_PREFERENCE_COOKIE_NAME } = process.env;
+const FooterWithLanguageSelector = ({ authenticatedUser = null }) => {
   const contextValue = useMemo(() => ({
-    authenticatedUser: null,
+    authenticatedUser,
     config: {
+      ENABLE_FOOTER_LANG_SELECTOR: true,
+      LANGUAGE_PREFERENCE_COOKIE_NAME,
```

Comment:
This should be implemented as part of https://github.com/openedx/frontend-platform/pull/786.  This PR should only be concerned with the UI elements.